core/src/consensus/mod.rs

```diff
@@ -265,7 +239,6 @@ pub trait ConsensusEngine: Sync + Send {
         header.hash()
     }
 
-    /// In PoW consensus, the higher scored block becomes the best block.
```

Comment:
We may remove the "score" in the header. When using the Tendermint consensus algorithm, we can't define a valid "score" value. A finalized block must have a higher score than a proposal block. Since the content of a block is not changed whether it is finalized or not. The score always has a weird value.

Reply:
I made an issue for it.